plugins/Feedback/API.php

```diff
@@ -31,11 +31,11 @@ class API extends \Piwik\Plugin\API
      * config: "feedback_email_address".
      *
      * @param string|null $featureName  The name of a feature you want to give feedback to.
-     * @param int         $like         Whether you like the feature or not
+     * @param string|null $like         Whether you like the feature or not
```

Comment:
I guess this is a bit strange to use `string|null` and then set the default to `$like = false`.